changelog/__import.dd

```diff
@@ -0,0 +1,21 @@
+Add ability to import modules to ImportC
+
+ImportC can now import modules with the __import keyword, with the C code:
+
+---
+__import core.stdc.stdarg; // import a D source file
+__import test2;		   // import an ImportC file
```

Comment:
So we have `betterC` as crippled D and `ImportC` as C on steroids? ;)

Reply:
yup! Use ImportC, and you won't need `dtoh` anymore!

Reply:
> yup! Use ImportC, and you won't need `dtoh` anymore!

`dtoh` exports a templated struct to mimic a D slice, how does this replicate that?